### PR TITLE
Update copyright headers to 2015

### PR DESCRIPTION
**Pull request by maflcko**
This can be merged into 0.12 and master without conflicts or rebase.

The first commit will fix the python helper script introduced in #3646 to no longer remove previous copyright years.

### CODE REVIEW

File: src/test/bip32_tests.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2013 The Bitcoin Core developers
+// Copyright (c) 2015 The Bitcoin Core developers
```
Comment (MarcoFalke) [inline src/test/bip32_tests.cpp]: This should update to `2013-2015`, I guess.

**Comment by jonasschnelli** [comment]
Does it only change files that where changed during that year? Or does the script just changes every copyright information?

**Comment by fanquake** [comment]
It should only change files that have been modified.

**Comment by MarcoFalke** [comment]
Right, it only updates files where `GitAuthorDate` of the last commit to this file contains `2015`.

**Comment by jonasschnelli** [comment]
Concept ACK.

File: src/test/bip32_tests.cpp (hunk 1 above)
Comment (MarcoFalke) [inline src/test/bip32_tests.cpp]: Updated OP

**Pull request merged**